### PR TITLE
fix: (Core) fix Card accessibility issue

### PR DESCRIPTION
**Pull request by DeepakSap14**
#### Please provide a link to the associated issue.
#3545 #3546 #3547 #3560 #3811 #3813 #3817 #3818

#### Please provide a brief summary of this pull request.
1. Fixes tab issues
2. screen reader issues

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: apps/docs/src/app/core/component-docs/fixed-card-layout/examples/default/fixed-card-layout-examples.component.html [hunk 1]
```diff
@@ -89,7 +90,7 @@ <h4 fd-card-second-subtitle>Q1, 2018</h4>
                     <h2 fd-card-title>Table card</h2>
                     <span fd-card-counter>3 of 10</span>
                 </fd-card-header>
-                <fd-card-content>
+                <fd-card-content [attr.tabIndex]="0">
```
Comment (Betrozov) [inline apps/docs/src/app/core/component-docs/fixed-card-layout/examples/default/fixed-card-layout-examples.component.html:93]: `[attr.tabIndex]="0"` => `tabIndex="0"`

File: apps/docs/src/app/core/component-docs/fixed-card-layout/examples/default/fixed-card-layout-examples.component.html [hunk 2]
```diff
@@ -100,7 +101,7 @@ <h2 fd-card-title>Table card</h2>
                             </tr>
                         </thead>
                         <tbody fd-table-body>
-                            <tr fd-table-row [main]="true" [activable]="true">
+                            <tr fd-table-row [main]="true" [activable]="true" [attr.tabIndex]="0">
```
Comment (Betrozov) [inline apps/docs/src/app/core/component-docs/fixed-card-layout/examples/default/fixed-card-layout-examples.component.html:104]: the same

File: apps/docs/src/app/core/component-docs/fixed-card-layout/examples/disabled-drag-drop/fixed-card-layout-disabled-drag.component.html [hunk 3]
```diff
@@ -76,7 +77,7 @@ <h4 fd-card-second-subtitle>Q1, 2018</h4>
                 <h2 fd-card-title>Table card</h2>
                 <span fd-card-counter>3 of 10</span>
             </fd-card-header>
-            <fd-card-content>
+            <fd-card-content [attr.tabIndex]="0">
```
Comment (Betrozov) [inline apps/docs/src/app/core/component-docs/fixed-card-layout/examples/disabled-drag-drop/fixed-card-layout-disabled-drag.component.html:80]: the same

File: apps/docs/src/app/core/component-docs/fixed-card-layout/examples/mobile/fixed-card-layout-mobile-examples.component.html [hunk 4]
```diff
@@ -82,7 +84,7 @@ <h4 fd-card-second-subtitle>Q1, 2018</h4>
                         <h2 fd-card-title>Table card</h2>
                         <span fd-card-counter>3 of 10</span>
                     </fd-card-header>
-                    <fd-card-content>
+                    <fd-card-content [attr.tabIndex]="0">
```
Comment (Betrozov) [inline apps/docs/src/app/core/component-docs/fixed-card-layout/examples/mobile/fixed-card-layout-mobile-examples.component.html:87]: the same

File: libs/core/src/lib/card/card.component.html [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <ng-container *ngIf="!isLoading; else loaderTmpl">
     <!-- Badge -->
-    <div *ngIf="badge" class="fd-badge">{{ badge }}</div>
+    <div *ngIf="badge" class="fd-badge" [attr.tabindex]="0">{{ badge }}</div>
```
Comment (Betrozov) [inline libs/core/src/lib/card/card.component.html:3]: the same

File: libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.html [hunk 6]
```diff
@@ -4,7 +4,7 @@
         <ng-container *ngFor="let card of column">
             <div class="fd-fixed-card-group--card" cdkDrag [cdkDragStartDelay]="dragStartDelay"
                 cdkDragPreviewClass="fd-fixed-card-group--card-preview" [cdkDragData]="card"
-                (cdkDragDropped)="dragDrop($event)">
+                (cdkDragDropped)="dragDrop($event)" [attr.tabindex]="0">
```
Comment (Betrozov) [inline libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.html:7]: the same

**Review by Betrozov — CHANGES_REQUESTED** [review]

**Review by InnaAtanasova — COMMENTED** [review]
Same comment as Yury

File: apps/docs/src/app/core/component-docs/card/examples/card-kpi-example.component.html [hunk 7]
```diff
@@ -24,7 +26,7 @@ <h2 fd-card-title>Failure Breakdown</h2>
 
         <h4 fd-card-second-subtitle>Q1, 2019</h4>
     </fd-card-header>
-    <fd-card-content>
+    <fd-card-content tabindex="0">
```
Comment (InnaAtanasova) [inline apps/docs/src/app/core/component-docs/card/examples/card-kpi-example.component.html:29]: Typo

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
is it `tabindex` or `tabIndex`? You are using both

**Review by Betrozov — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

File: apps/docs/src/app/core/component-docs/card/examples/bar-chart-list-card/bar-chart-list-card-example.component.html [hunk 8]
```diff
@@ -4,7 +4,7 @@ <h2 fd-card-title>Bar Chart List Card</h2>
         <span fd-card-counter>4 of 10</span>
     </fd-card-header>
     <fd-card-content>
-        <ul class="fd-list fd-list--byline">
+        <ul class="fd-list fd-list--byline" [attr.tabindex]="0">
```
[the remark this answers is not in the document]
Comment (DeepakSap14) [reply apps/docs/src/app/core/component-docs/card/examples/bar-chart-list-card/bar-chart-list-card-example.component.html]: using tabindex now. removed Typo.
replcaed [attr.tabindex] with "tabindex".

**Pull request merged**